docs/README.md

```diff
@@ -0,0 +1,46 @@
+## Hubble Web Application
+
+This web application is built as a [Jekyll](https://jekyllrb.com/) site and is intended to be hosted using [GitHub Pages](https://pages.github.com/).
+
+## Overview
+
+### Charts
+
+Each `.html` page in this directory contains one or more charts. Each chart is defined as an `<svg>` element with a few special attributes that define their behaviour:
+
+1. a `data-url` attribute referencing a `.tsv` file generated by the [updater component](../updater/README.md) - this is a relative URL that directs the chart where to ask for data from.
+1. a `class` attribute defining the kind of chart that will be rendered. Acceptable values are `chart`, `table`, and `collaboration`.
+1. a `data-type` _only applicable if the `class` is set to `chart`_; attribute referencing the sub-type of chart that will be rendered. Acceptable values are `history` and `list`.
+
+For details on how each kind of chart is rendered, take a look at the [`charts.js` file](assets/js/charts.js).
+
+## Development
+
+To run the site locally, you will need to install [Jekyll](https://jekyllrb.com/). Then change the `_config.yml` file and set the `dataURL` to `http://127.0.0.1:4000/demo-data`. Finally, run `jekyll serve`,
+and load http://127.0.0.1:4000 in your browser.
+
+## Tests
+
+This web application uses a framework called [Karma](https://karma-runner.github.io) as a test runner (a way to run JavaScript in different browser contexts), and [Jasmine](https://jasmine.github.io)
+as the testing / assertion library. The tests are implemented as `.js` files under the `spec/` folder, and as much as possible mirror the filenames of JavaScript source files.
+
+### Requirements
+
+- You will need a recent version of [node.js](https://nodejs.org) installed.
+- The tests run in a headless instance of the **Google Chrome** browser, so you will need a recent version of Google Chrome installed.
+- Install any needed dependencies by running `npm install` from this folder.
+
+### Running the Tests
+
+To run the tests once:
+
+    $ npm test
+
+During local development, it may be nice to keep the test runner running in the background as one makes changes to the JavaScript source files (under `assets/js/`), or the test files (under `spec/`) so as to
+get immediate feedback. This mode will re-run the tests every time a source or test file changes. To run in this mode:
+
+    $ npm test -- --no-single-run --auto-watch
+
+### Linting
+
+    $ npm run lint
```

Comment:
The linter complains it can't find a config file. I guess this is expected?

Reply:
Ah yes, I have it ready to go but I was going to wait until the tests were merged before adding all the various bits. I will remove these instructions for now until I put together the full additions for linting. I'll also rebase with latest master.